### PR TITLE
cloud: add release notes for 1213

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/release-notes-2022.md [hunk 1]
```diff
@@ -8,6 +8,43 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+- Support using Changefeed to stream data for Dedicated Tier.
+
+    - Support streaming data change logs to MySQL.
+
+      When data is migrated from MySQL/Aurora to TiDB, it is often necessary to use MySQL as a stand-by database to prevent unexpected data migration problems. In this case, you can use MySQL sink to stream data from TiDB to MySQL.
+
+    - Support streaming data change logs to Apache Kafka (Beta).
+
+      Streaming TiDB data to the message queue is a very common requirement for data integration scenarios. You can use Kafka sink to realize integration with other data processing systems (such as snowflake), or support business consuming.
```
Comment (Oreoxmt) [inline tidb-cloud/release-notes-2022.md:29]:
```suggestion
      Streaming TiDB data to the message queue is a very common requirement for data integration scenarios. You can use Kafka sink to realize integration with other data processing systems (such as Snowflake), or support business consuming.
```

**Review by IANTHEREAL — APPROVED** [review]

**Review by zhoubasten — APPROVED** [review]
LGTM

File: tidb-cloud/release-notes-2022.md [hunk 2]
```diff
@@ -8,6 +8,43 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    For more information, refer to [Changefeed Overview](https://docs.pingcap.com/tidbcloud/changefeed-overview).
+
+- Organization owners can edit the organization's name.
```
Comment (Oreoxmt) [inline tidb-cloud/release-notes-2022.md:33]:
```suggestion
- Organization owners can edit the organization's name in **Organization Settings**.
```

**Review by langyuezhang — APPROVED** [review]
LGTM

File: tidb-cloud/release-notes-2022.md [hunk 3]
```diff
@@ -8,6 +8,43 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+      When data is migrated from MySQL/Aurora to TiDB, it is often necessary to use MySQL as a stand-by database to prevent unexpected data migration problems. In this case, you can use MySQL sink to stream data from TiDB to MySQL.
+
+    - Support streaming data change logs to Apache Kafka (Beta).
+
+      Streaming TiDB data to the message queue is a very common requirement for data integration scenarios. You can use Kafka sink to realize integration with other data processing systems (such as Snowflake), or support business consuming.
+
+    For more information, refer to [Changefeed Overview](https://docs.pingcap.com/tidbcloud/changefeed-overview).
+
+- Organization owners can edit the organization's name in **Organization Settings**.
```
[the remark this answers is not in the document]
Comment (ran-huang) [reply tidb-cloud/release-notes-2022.md:33]: No. This feature is too obvious and does not need any doc.

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**